test/llvm/alignment/check-align-padding.bad

```diff
@@ -1,7 +1 @@
-%int32int64_chpl = type { i32, i64 }
```

Comment:
Can you split this test into 2 cases - one where it compiles and we the alignment is OK at runtime - and another where we see this error?

Reply:
I split it into two tests as you requested.

Reply:
Shouldn't this .bad file go away?

Reply:
The .bad file matches the current behavior of the .future, so I think it should stay.